### PR TITLE
Pull request for Artillery Genius to Switchwire conversion

### PR DESCRIPTION
**Pull request by salveoo**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Artillery Genius to Voron Switchwire conversion
#### How was it tested? 
This mod has been built and got a serial (SW #193)
#### Any background context you want to provide?
-
#### Screenshots (if appropriate)
-
#### Further notes
-

### CODE REVIEW

**Comment by bdbell** [comment]
Two things:
1. All files names need to have the spaces replaced with underscores, and I would prefer the parenthesis be removed.
2. Some of the STL's are in an incorrect orientation for printing.

**Comment by salveoo** [comment]
It should be all good now

**Review by cbc02009 — CHANGES_REQUESTED** [review]

**Comment by salveoo** [comment]
Done!

File: printer_mods/Salveo/ArtilleryGenius_SW_conversion/CFG/printer.cfg [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+screw4: 40,190
+
+[virtual_sdcard]
+path: /home/pi/gcode_files
+
+[stepper_x]
+step_pin: ar54
+dir_pin: !ar55
+enable_pin: !ar38
+step_distance: .0125
```
Comment (cbc02009) [inline printer_mods/Salveo/ArtilleryGenius_SW_conversion/CFG/printer.cfg:30]: Klipper no longer uses step_distance, so you'll need to update this config. there's a reference one here: https://github.com/VoronDesign/Voron-Switchwire/pull/31/files you can look at.

**Review by cbc02009 — CHANGES_REQUESTED** [review]
[This STL](https://github.com/VoronDesign/VoronUsers/blob/6cd824e672ae302b2bc5e8381122382535efda65/printer_mods/Salveo/ArtilleryGenius_SW_conversion/STL/GCODE_BUTTONS_MOD/top.stl) appears to be upside down.

[This STL](https://github.com/VoronDesign/VoronUsers/blob/6cd824e672ae302b2bc5e8381122382535efda65/printer_mods/Salveo/ArtilleryGenius_SW_conversion/STL/CABLE_CHAINS/chain_endlink.stl) is likely going to have issues with printability. It might be a good idea to add integrated supports so that it can be printed on the bottom like the [panzer chains](https://github.com/VoronDesign/VoronUsers/blob/master/legacy_printers/printer_mods/Slidr/PanzerChain/XY/XY_extrusion_anchor_x2.stl).

Also see the other comment about step_distance.

**Comment by cbc02009** [comment]
Closing due to inactivity.

**Pull request closed without merging**